app/build.gradle

```diff
@@ -2,11 +2,12 @@ apply plugin: 'com.android.application'
 apply plugin: 'kotlin-android'
 
 android {
-	namespace 'de.markusfisch.android.binaryeye'
```

Comment:
Why did you put this line at the end of the block? For me, defining a namespace is something I like to do next. Are there any reasons I haven't thought of?

Reply:
Your argument makes sense. My personal preference is at the bottom, but I will move it up.